review-cask-pr/review.rb

```diff
@@ -177,8 +177,10 @@ def review_diff(pr, author_is_member: false)
 
       puts message
 
-      puts "::set-output name=event::#{event}"
-      puts "::set-output name=message::#{GitHub::Actions.escape(message)}"
+      File.open(ENV["GITHUB_OUTPUT"], "a") do |f|
+        f.puts("event=#{event}")
+        f.puts("message=#{GitHub::Actions.escape(message)}")
```

Comment:
Escaping works differently now so this doesn't work. The new way to do it is:

```
{name}<<{delimiter}
{value}
{delimiter}
```

For security reasons `delimiter` must be randomly generated and unique to each run (e.g. UUID v4). It should also be something that doesn't appear in the `message`. It may be worth having a new method in `Homebrew/brew`.

See https://github.com/actions/toolkit/blob/b00a9fd033f4b30f2355acd212f531ecbbb9b38f/packages/core/src/file-command.ts#L27-L47 for an example implementation in JavaScript.